### PR TITLE
Updated configs

### PR DESCRIPTION
**Pull request by timbrinded**
- Updated `acala` and `karura` configs to latest client versions
- ~~Added `yarn` utility scripts~~

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,9 +22,12 @@
   "scripts": {
     "build": "tsc --build tsconfig.json",
     "lint": "polkadot-dev-run-lint",
-    "start": "ts-node src/index.ts"
+    "start": "ts-node src/index.ts",
+    "generate:acala": "npx parachain-launch generate --config=configs/acala.yml --y",
+    "generate:karura": "npx parachain-launch generate --config=configs/karura.yml --y"
   },
   "dependencies": {
+    "@open-web3/parachain-launch": "^1.2.0",
```
Comment (xlc) [inline package.json:30]: why depends on self?

File: package.json [hunk 2]
```diff
@@ -22,9 +22,12 @@
   "scripts": {
     "build": "tsc --build tsconfig.json",
     "lint": "polkadot-dev-run-lint",
-    "start": "ts-node src/index.ts"
+    "start": "ts-node src/index.ts",
+    "generate:acala": "npx parachain-launch generate --config=configs/acala.yml --y",
```
Comment (xlc) [inline package.json:26]: let’s move those to here https://github.com/AcalaNetwork/Acala/blob/master/launch/package.json

Comment (timbrinded) [reply package.json]: Done

File: package.json (hunk 1 above)
Comment (timbrinded) [reply package.json]: This was so i could run it with `npx` but you're right, should have just run the `parachain-launch.js` locally. In any case this is removed now given the yarn scripts have been removed.

**Review by xlc — APPROVED** [review]

**Pull request merged**